yarn.lock

```diff
@@ -10865,15 +10865,21 @@ jsonpath-plus@^6.0.1:
   resolved "https://registry.yarnpkg.com/jsonpath-plus/-/jsonpath-plus-6.0.1.tgz#9a3e16cedadfab07a3d8dc4e8cd5df4ed8f49c4d"
   integrity sha512-EvGovdvau6FyLexFH2OeXfIITlgIbgZoAZe3usiySeaIDm5QS+A10DKNpaPBBqqRSZr2HN6HVNXxtwUAr2apEw==
 
-jsonwebtoken@^8.5.1, jsonwebtoken@^9.0.0:
```

Comment:
should yarn.lock be ignored?

Reply:
No, package-lock.json should be commited. The lock file changed due to changes in `package.json`. (see comment above)